src/main/java/io/dropwizard/actors/RabbitmqActorBundle.java

```diff
@@ -34,4 +42,21 @@ public void initialize(Bootstrap<?> bootstrap) {
     }
 
     protected abstract RMQConfig getConfig(T t);
+
+    /**
+     * Provides metric registry for instrumenting RMQConnection. If method returns null, metrics collector for
+     * RabbitMQ connection is not initialized.
+     */
+
+    protected MetricRegistry getMetricRegistry(T t) {
```

Comment:
Pass in Environment here as well and return environment.metrics() by default. They should always be sending metrics.